src/autotuner/genetic_tuning_harness.cc

```diff
@@ -234,18 +234,18 @@ bool GeneticTunerHarness::warmupOrPrune(
             CHECK(info);
             USING_MAPPING_SHORT_NAMES(BX, BY, BZ, TX, TY, TZ);
             auto& exec = info->exec;
-            auto block = exec.block();
+            auto block = static_cast<CudaTcExecutor&>(*exec).block();
             auto nThreads = TX.mappingSize(block) * TY.mappingSize(block) *
                 TZ.mappingSize(block);
-            auto grid = exec.grid();
+            auto grid = static_cast<CudaTcExecutor&>(*exec).grid();
```

Comment:
I know Nicolas doesn't like getters, but surely
it's better to perform this `static_cast` in at most one place.